app/toolbox-server/Main.hs

```diff
@@ -46,21 +48,47 @@ import Toolbox.Server.Types
 import Toolbox.Worker (moduleGraphWorker)
 import Prelude hiding (div)
 
-newtype Options = Options {optSocketFile :: FilePath}
+data CLIMode
+  = Online FilePath
+  | View FilePath
 
-optsParser :: OA.ParserInfo Options
+onlineMode :: OA.Mod OA.CommandFields CLIMode
+onlineMode =
+  OA.command
+    "online"
+    ( OA.info
```

Comment:
Not a requested change, but curious to know the decision to use `(` `)` over `$` here for brevity?

Reply:
no reason here :-D  In many cases, it is often a remnant when additional argument for the function existed and later removed, but this is not the case. I will replace that with dollar for a better look. Thanks.